student-manager-maven/src/main/java/com/demo/vews/Main.java

```diff
@@ -0,0 +1,123 @@
+package com.demo.vews;
+
+import java.util.Scanner;
+
+import com.demo.controller.RoomController;
+import com.demo.controller.StudentController;
+import com.demo.model.Person;
+import com.demo.model.Room;
+import com.demo.model.StudentBlockA;
+import com.demo.model.StudentBlockB;
+import com.demo.model.StudentBlockC;
+
+public class Main {
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		StudentController studentController = new StudentController();
+		RoomController roomController = new RoomController();
+		while (true) {
+			System.out.println("Application Manager Student");
+			System.out.println("Enter 1: To insert student");
+			System.out.println("Enter 2: To show information of student: ");
+			System.out.println("Enter 3: To search student by id");
+			System.out.println("Enter 4: To insert room");
+			System.out.println("Enter 5: To show information of room: ");
+			System.out.println("Enter 6: To exit:");
+			String line = scanner.nextLine();
+			switch (line) {
+			case "1": {
+				System.out.println("Enter a: to insert Student Block A");
+				System.out.println("Enter b: to insert Student Block B");
+				System.out.println("Enter c: to insert Student Block C");
+				String type = scanner.nextLine();
+				switch (type) {
+				case "a": {
+					studentController.add(createPerson(scanner, "a"));
+					break;
+
```

Comment:
unused break line again. Please check the other place!

Reply:
Update